charts/rancher-webhook/values.yaml

```diff
@@ -22,4 +22,5 @@ preDelete:
 
 # tolerations for the webhook deployment. See https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/ for more info
 tolerations: []
-nodeSelector: {}
\ No newline at end of file
+nodeSelector: {}
+priorityClassName: ""
```

Comment:
Nit: If you end up needing to edit this, would you mind adding a comment indicating what this is for? I'm not too attached to it since it maps exactly to a k8s value, but I think ti would be nice to mirror what we do for tolerations if we can.